erpnext/buying/doctype/request_for_quotation/request_for_quotation.js

```diff
@@ -326,6 +326,11 @@ erpnext.buying.RequestforQuotationController = erpnext.buying.BuyingController.e
 				d.show();
 			}, __("Get items from"));
 
+			//Link Material Requests
```

Comment:
```suggestion
			// Link Material Requests
```
Sorry, couldn't resist.